app/models/conversation.rb

```diff
@@ -26,6 +26,10 @@ class Conversation < ApplicationRecord
 
   validate :users_uniqueness, on: :create
 
+  def other_user(user)
+    users.where.not(id: user.id).first
```

Comment:
```suggestion
    users.where.not(id: user.id)&.first
```

Reply:
Whats & for?

Reply:
It's a way to avoid a null error. Take a look at [this](https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/).